packages/mgt-components/src/components/mgt-file-list/mgt-file-list.scss

```diff
@@ -9,23 +9,23 @@
 @import '../../styles/shared-styles.scss';
 @import './mgt-file-list.theme.scss';
 
-$file-item-background-color--hover: set-var(list__item__background-color--hover, $theme-default, $file-list);
-$file-item-background-color--active: set-var(list__item__background-color--active, $theme-default, $file-list);
-$background-color: set-var(list__background-color, $theme-default, $file-list);
-$file-list-border-radius: var(--file-list-border-radius, 4px);
-$file-list-box-shadow: var(
-  --file-list-box-shadow,
-  0px 3.2px 7.2px rgba(0, 0, 0, 0.132),
-  0px 0.6px 1.8px rgba(0, 0, 0, 0.108)
-);
+$file-list-border-radius: var(--file-list-border-radius, 8px);
```

Comment:
is the change from 2px to 8px intentional here?

Reply:
Yes. The figma design set it to 8px.